portfolio/src/main/webapp/hello.html

```diff
@@ -1,6 +1,21 @@
+
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>(Hopefully) a successful "hello" page)</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    
 So I'm hoping that by creating a new .html file called "hello",
 <br>
 the "hello" hyperlink will now work
```

Comment:
This line is not part of a specific element.

Reply:
Do you mean <code>\<link rel="stylesheet" href="style.css"></code>? If so, I just copied that header from the example webpage given; if I don't use it, then do I just not include it?

Reply:
I was referring to:

```
<body>
So I'm hoping that by creating a new .html file called "hello",
...
```

The text following the `<body>` tag isn't in a block layout context. It's sort of floating there. It will work but isn't good structure.